hnn_core/viz.py

```diff
@@ -553,16 +553,23 @@ def plot_cell_morphology(cell, ax, show=True):
         plt.figure()
         ax = plt.axes(projection='3d')
 
-    ax.set_ylim((-100, 1200))
-    ax.set_xlim((-250, 150))
+    # Cell is in XZ plane
+    ax.set_xlim((cell.pos[1] - 250, cell.pos[1] + 150))
+    ax.set_zlim((cell.pos[2] - 100, cell.pos[2] + 1200))
+
     for sec_name, p_sec in cell.p_secs.items():
         linewidth = _linewidth_from_data_units(ax, p_sec['diam'])
         sec_pts = p_sec['sec_pts']
-        xs = [pt[0] for pt in sec_pts]
-        ys = [pt[1] for pt in sec_pts]
-        zs = [pt[2] for pt in sec_pts]
+        xs, ys, zs = list(), list(), list()
+        for pt in sec_pts:
+            dx = cell.pos[0] - cell.p_secs['soma']['sec_pts'][0][0]
+            dy = cell.pos[1] - cell.p_secs['soma']['sec_pts'][0][1]
+            dz = cell.pos[2] - cell.p_secs['soma']['sec_pts'][0][2]
```

Comment:
this is probably overkill but it helped me verify what I was doing is correct. You can potentially pass `ax` and plot all the cells in one plot this way.

Reply:
Good call.  This is a good sanity check.